micro-ui/web/micro-ui-internals/packages/modules/core/src/pages/employee/QuickStart/QuickSetup.js

```
@@ -1,9 +1,41 @@
+import { ArrowForward } from "@egovernments/digit-ui-svg-components";
 import { CardSubHeader, CardSectionHeader, BreakLine, CardSectionSubText } from "@egovernments/digit-ui-react-components";
-import { Link } from "react-router-dom";
-import FaqComponent from "../../citizen/FAQs/FaqComponent";
+
+const FaqComponent = (props) => {
+  const { question, answer,type,actions,content, lastIndex } = props;
+  const [isOpen, toggleOpen] = useState(false);
+  const { t } = useTranslation();
+  const ListTag = type === "number" ? "ol" : "ul";
+  return (
+    <div className="faqs border-none" onClick={() => toggleOpen(!isOpen)}>
```

Biome

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)